dep.go

```diff
@@ -131,6 +133,13 @@ func (g *Godeps) Load(pkgs []*Package) error {
 			err1 = errors.New("error loading dependencies")
 			continue
 		}
+		if saveP {
+			fmt.Printf("godep: Would you like to index %s ?\n[Y/n]:", pkg.ImportPath)
+			text, _ := reader.ReadString('\n')
+			if text[0] == 'n' {
+				continue
+			}
```

Comment:
This should be

```go
if len(text) < 1 || text[0] != 'y' {
        continue
}
```

Prevent out-of-bounds index panic and be a little stricter about the input.